code/game/objects/structures/window.dm

```diff
@@ -444,7 +444,7 @@
 /obj/structure/window/plasma/reinforced
 	name = "reinforced plasma window"
 	desc = "A window made out of a plasma-silicate alloy and a rod matrice. It looks hopelessly tough to break and is most likely nigh fireproof."
-	icon_state = "rplasmawindow"
+	icon_state = "plasmawindow"
```

Comment:
This is the reinforced plasma window sprite location. No idea why I named it that, I made the PR while I was awake for 24 hours

https://github.com/tgstation/tgstation/blob/master/icons/obj/smooth_structures/rplasma_window.dmi